### PR TITLE
Fennec weekly update 15092015

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: mobile/android/chrome/aboutAccounts.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+relinkDenied.message = Już zalogowano do Sync
+relinkDenied.openPrefs = USTAWIENIA
```
Comment (splewako) [inline mobile/android/chrome/aboutAccounts.properties:6]: This doesn't look right at all - are you sure "PREFS" (?!) is translated as "USTAWIENIA" in Polish Android UI?

File: mobile/android/base/sync_strings.dtd [hunk 2]
```diff
@@ -193,6 +193,7 @@ the two uses differently. -->
 
 <!ENTITY fxaccount_status_header2 "Konto Firefoksa">
 <!ENTITY fxaccount_status_signed_in_as 'Zalogowano jako'>
+<!ENTITY fxaccount_status_manage_account 'Zarządzaj kontem'>
```
Comment (splewako) [inline mobile/android/base/sync_strings.dtd]: Use double quotes for consistency in future please.

**Comment by splewako** [comment]
https://hg.mozilla.org/l10n-central/pl/pushloghtml?changeset=ba0e9023f516 and https://github.com/aviarypl/mozilla-l10n-pl/compare/91b4d323345c...36add2f367d5 but I'm leaving this pull request open for `relinkDenied.openPrefs`

File: mobile/android/chrome/aboutAccounts.properties (hunk 1 above)
Comment (piotrdrag) [inline mobile/android/chrome/aboutAccounts.properties:6]: Personally, I would prefer "Preferencje", but indeed there are only "Ustawienia" on Android.

File: mobile/android/base/android_strings.dtd [hunk 3]
```diff
@@ -508,28 +475,23 @@ it is applicable. -->
...
 
 <!ENTITY searchable_description "Zakładki i historia">
 
-<!-- Updater notifications -->
 <!ENTITY updater_start_title2 "&brandShortName;: dostępne są nowe aktualizacje">
 <!ENTITY updater_start_select2 "Dotknij, aby zaktualizować">
 
 <!ENTITY updater_downloading_title2 "Pobieranie: &brandShortName;">
-<!ENTITY updater_downloading_title_failed2 "Pobieranie nie udało się">
+<!ENTITY updater_downloading_title_failed2 "Pobieranie się nie udało">
```
Comment (splewako) [inline mobile/android/base/android_strings.dtd:488]: For me [it is not that obvious](https://github.com/piotrdrag/mozilla-l10n-pl/commit/af9cbfe988226b34a860ed0bd937eecc50b7f103), see Aviary.pl bugs [4769](http://bugs.aviary.pl/show_bug.cgi?id=4769), [2645](http://bugs.aviary.pl/show_bug.cgi?id=2645), [2811](http://bugs.aviary.pl/show_bug.cgi?id=2811) and [1074](http://bugs.aviary.pl/show_bug.cgi?id=1074) among others.

Comment (piotrdrag) [inline mobile/android/base/android_strings.dtd:488]: Right, those. How can I forget. But could we agree that "się nie powiodło" is ok, or at least not worse than "nie powiodło się"?

Comment (splewako) [inline mobile/android/base/android_strings.dtd:488]: Sure, just a nit/reminder - you have the last word over the files you are taking care of.

Comment (piotrdrag) [inline mobile/android/base/android_strings.dtd:488]: So let's do it. :)

**Comment by piotrdrag** [comment]
Is there anything else blocking the push?

**Comment by splewako** [comment]
No, nothing, I will do that later today probably

**Comment by piotrdrag** [comment]
I thought I missed something. Thanks!

**Comment by splewako** [comment]
https://hg.mozilla.org/l10n-central/pl/pushloghtml?changeset=fab1a599d24f

Thank you for the patches

**Pull request closed without merging**